frontend-react/src/oktaConfig.ts

```diff
@@ -18,30 +24,83 @@ const oktaAuthConfig: OktaAuthOptions = {
     services: {
         autoRenew: false,
     },
+    async transformAuthState(oktaAuth, authState) {
+        let finalAuthState: AuthState = structuredClone(authState);
+
+        // Prevent pulling incorrect token from a different okta environment
+        if (
+            authState.isAuthenticated &&
+            authState.accessToken?.claims.iss !== OKTA_ISSUER
+        ) {
+            oktaAuth.clearStorage();
+            finalAuthState = {
+                ...authState,
+                accessToken: undefined,
+                idToken: undefined,
+                refreshToken: undefined,
+                isAuthenticated: false,
+            };
+        }
+
+        return finalAuthState;
+    },
 };
+const OKTA_AUTH = new OktaAuth(oktaAuthConfig);
 
 const oktaSignInConfig: WidgetOptions = {
+    ...sharedConfig,
     logo: "/assets/cdc-logo.svg",
     language: "en",
     features: {
         registration: false, // Disable self-service registration flow
         rememberMe: false, // Setting to false will remove the checkbox to save username
-        router: true, // Leave this set to true for the API demo
-        // TODO: bring up enabling these once we have OktaPreview
-        // securityImage: true, // helps prevent spoofing, may require CORS to allow it through.
-        // mfaOnlyFlow: true, // government requires mfa
+        router: false, // Leave this set to true for the API demo
```

Comment:
Should this be true?

Reply:
With it enabled it changes the URL in the browser during the login process (/signin/*) which is a conflict with our app router. I haven't found any reason why we would want this.

Reply:
maybe add this comment to the code so that others understand better.